src/ai/configuration.cpp

```diff
@@ -289,6 +290,9 @@ void configuration::expand_simplified_aspects(side_number side, config &cfg) {
 		}
 		if (aiparam.has_attribute("engine")) {
 			engine = aiparam["engine"].str();
+			if(engine == "fai") {
+				deprecated_message("FormulaAI", DEP_LEVEL::FOR_REMOVAL, "1.17", "FormulaAI is slated to be removed. Use equivalent Lua AIs instead");
```

Comment:
@Pentarctagon what's our plan and timescale for removing FormulaAI?

Reply:
I remember someone (either Pentarctagon or mattsc) saying the plan is to remove in 1.17, which is why I've added these messages for backporting.

Reply:
The plan is for it to be removed in 1.17.